tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -955,9 +955,16 @@ def validate_layout(layout: str):
     :param layout: string containing layout
     :raises: if layout is incorrect
     """
-    valid_layout_re = re.compile(r'\[?[^\[\]\(\)\s]*\]?')
-    if layout and not valid_layout_re.fullmatch(layout):
-        raise Error('Invalid layout parsed: {}'.format(layout))
+    error_msg = 'Invalid layout parsed: {}'.format(layout)
+    if layout:
+        incorrect_brackets = xor(layout[0] == '[', layout[-1] == ']')
+        if incorrect_brackets or layout[-1] == '-':
+            error_msg += ', did you forget quotes?'
+        else:
+            valid_layout_re = re.compile(r'\[?[^\[\]\(\)\-\s]*\]?')
+            if valid_layout_re.fullmatch(layout):
+                return
+        raise Error(error_msg)
```

Comment:
Would be really nice to cover this function by tests.

Reply:
There are tests that cover this function. This changes only improve error log, I don't think we need explicit tests for that